src/webgpu/api/operation/buffers/mapping_test.ts

```diff
@@ -1,18 +1,24 @@
+import { assert } from '../../../../common/framework/util/util.js';
 import { GPUTest } from '../../../gpu_test.js';
 
 export class MappingTest extends GPUTest {
-  checkMapWrite(buffer: GPUBuffer, mappedContents: ArrayBuffer, size: number): void {
+  checkMapWrite(
+    buffer: GPUBuffer,
+    offset: number,
+    mappedContents: ArrayBuffer,
+    size: number
+  ): void {
     this.checkMapWriteZeroed(mappedContents, size);
 
     const mappedView = new Uint32Array(mappedContents);
     const expected = new Uint32Array(new ArrayBuffer(size));
-    this.expect(mappedView.byteLength === size);
+    assert(mappedView.byteLength === size);
```

Comment:
Why does this need to be an assert instead of an expect? If the length is wrong, is the rest of the error log too noisy?

Reply:
If the length is wrong it doesn't make sense to check the contents because the loop could go too long for `expected[i]`.

Reply:
(Could just let it hit an exception, or use min(mappedView.length, expected.length), though)

Reply:
ok yea, seems fine to `assert`